### ISSUE TITLE
Error running yarn: EPERM: operation not permitted on Windows 10

### ISSUE THREAD

**Issue by uluhonolulu** [electron-react-boilerplate/electron-react-boilerplate#2986]
## Prerequisites

- [x] Using yarn
- [x] Using an up-to-date [`main` branch](https://github.com/electron-react-boilerplate/electron-react-boilerplate/tree/main)
- [x] Using latest version of devtools. [Check the docs for how to update](https://electron-react-boilerplate.js.org/docs/dev-tools/)
- [x] Tried solutions mentioned in [#400](https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues/400)
- [ ] For issue in production release, add devtools output of `DEBUG_PROD=true yarn build && yarn start` **not a production issue**

## Expected Behavior

Running `yarn` should install all dependencies without errors

## Current Behavior

EPERM: operation not permitted

## Steps to Reproduce

Here is the full error message:
```
Error: EPERM: operation not permitted, symlink 'C:\Projects\Kopster\kopster_electron\build\app\node_modules' -> 'C:\Projects\Kopster\kopster_electron\src\node_modules'
    at Object.symlinkSync (fs.js:1210:3)
    at Object.<anonymous> (C:\Projects\Kopster\kopster_electron\.erb\scripts\/link-modules.js:8:6)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Module._compile (C:\Projects\Kopster\kopster_electron\node_modules\pirates\lib\index.js:99:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Object.newLoader [as .js] (C:\Projects\Kopster\kopster_electron\node_modules\pirates\lib\index.js:104:7)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12)
    at internal/main/run_main_module.js:17:47 {
  errno: -4048,
  syscall: 'symlink',
  code: 'EPERM',
  path: 'C:\\Projects\\Kopster\\kopster_electron\\build\\app\\node_modules',
  dest: 'C:\\Projects\\Kopster\\kopster_electron\\src\\node_modules'
}
```

## Possible Solution (Not obligatory)

Adding 'junction' as a third argument to https://github.com/electron-react-boilerplate/electron-react-boilerplate/blob/561569a9d89ff939b91b5f170007dc11f780d7c5/.erb/scripts/link-modules.js#L8 should probably fix the issue.

Workaround: Run as admin or switch to developer mode

## Context

Related: https://github.com/nodejs/node/issues/18518

## Your Environment

- Node version : 14
- electron-react-boilerplate version or branch : latest master
- Operating System and version : Windows 10

**Comment by lxfriday** [comment]
the same problem

**Comment by Wellbrito29** [comment]
I run in the admin mode and solved the problem

**Comment by caionakai** [comment]
your problem is because you are running yarn install on a terminal without admin permissions.
To solve it open CMD as admin and try yarn intall again

**Comment by uluhonolulu** [comment]
> your problem is because you are running yarn install on a terminal without admin permissions. To solve it open CMD as admin and try yarn intall again

Yes, this is what I suggested as a workaround. I think it should probably be added to the installation instructions.

**Comment by amilajack** [comment]
This should be fixed in #2948, which was recently released. Try testing out latest commit in main branch.
